src/lib.rs

```diff
@@ -80,32 +80,40 @@ impl<F: PrimeField, PC: MultiPC<F>, D: Digest> Marlin<F, PC, D> {
     /// Generate the index-specific (i.e., circuit-specific) prover and verifier
     /// keys. This is a deterministic algorithm that anyone can rerun.
     pub fn index<C: ConstraintSynthesizer<F>>(
-        universal_pk: &UniversalProverKey<F, PC>,
+        pp: &UniversalParams<F, PC>,
```

Comment:
```suggestion
        srs: &UniversalSRS<F, PC>,
```

Reply:
I changed the other function to use srs instead of pp as well.